bundle/DependencyInjection/Compiler/HttpCachePass.php

```diff
@@ -31,6 +31,7 @@ public function process(ContainerBuilder $container)
                     new Reference('ezplatform.http_cache.purge_client'),
                     new Reference('ezpublish.api.service.inner_content'),
                     new Reference('ezpublish.http_cache.event_dispatcher'),
+                    new Reference('ezpublish.api.repository'),
```

Comment:
this is also relevant for upcoming 6.7.7, 6.12.1 and 6.13.0.
Can code be done in different way to avoid having to set all arguments?

Reply:
https://github.com/ezsystems/ezpublish-kernel/commit/e778d2bd6ee2ba850f420f1fb729b832d8b6c342#diff-756d49898b2a4da2e9d185f1de4f4248

Reply:
I'll try to find a way. This is just a playground for now :)

Reply:
I don't see a way to fix this any other way :/

The problem is that by this point any reference to old `ezpublish.http_cache.purger.instant` is removed from the container and we simply do not know the list of arguments.